src/app/containers/author-form-dialog/index.tsx

```diff
@@ -155,6 +165,34 @@ export const AuthorFormDialog: React.FC<AuthorFormDialogProps> = (props) => {
         value={author.email || ''}
         onChange={handleFormChange}
       />
+      <FormControl variant="outlined" classes={{ root: classes.inputField }}>
+        <InputLabel shrink id="author-affiliations-label">
+          Affiliated Author
```

Comment:
Wrong Label, should be 'Competing interest'

Reply:
![image](https://user-images.githubusercontent.com/1397517/86597008-0f89ac80-bf93-11ea-9411-537b2ddba331.png)